Packages/Tracking/Core/Runtime/Scripts/Hands/CapsuleHand.cs

```diff
@@ -157,7 +157,18 @@ public override Hand GetLeapHand()
         /// <param name="hand"></param>
         public override void SetLeapHand(Hand hand)
         {
-            _hand = hand;
+            if (hand != null)
+            {
+                if (_hand == null)
+                {
+                    _hand = new Hand();
+                }
```

Comment:
Is this immediately being overwritten by the following line?

Reply:
It is but I believe its necessary as in order to do a "CopyFrom", _hand needs to be a Hand rather than null.